extensions/resource-deployment/notebooks/azuredb/create-sqldb.ipynb

```diff
@@ -0,0 +1,202 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Create Azure SQL Database\n",
+                "============================================\n",
+                "\n",
+                "Steps of this procedure include:\n",
+                "1. Connect to Azure subscription\n",
+                "1. Provision resource group for SQL Managed Instance\n",
+                "1. Create Sql Server\n",
+                "1. Provision firewall rules to allow access\n",
+                "1. Create Sql Database"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "6af59d69-ade7-480a-b33e-52a86fe5bfd3"
+            }
+        },
+		{
+            "cell_type": "markdown",
+            "source": [
+                "## Set variables"
+            ],
+            "metadata": {
+                "azdata_cell_guid": "b57c46c8-4a34-49af-9b62-aa5688a02002"
+            }
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Notebook Setup "
+            ],
+            "metadata": {
+                "azdata_cell_guid": "19ebf0fd-7010-4cd6-8bcd-d2f63dc75cfb"
+            }
+        },
+		{
+            "cell_type": "code",
+            "source": [
+                "import sys, os, json, time, string, random, subprocess\r\n",
+                "def run_command(command, json_decode = True, printOutput = True):\n",
+                "    print(command)\n",
+                "    process = subprocess.Popen(command.split(), stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)\n",
+                "    output, error = process.communicate()\n",
+                "    if process.returncode != 0: \n",
+                "        print(\"Process failed %d \\n%s\" % (process.returncode, error.decode(\"utf-8\")))\n",
+                "        raise Exception()\n",
+                "    if output:\n",
+                "        output = output.decode(\"utf-8\")\n",
+                "        if printOutput:\n",
+                "            print(output)\n",
+                "        try:\n",
+                "            return json.loads(output)\n",
+                "        except:\n",
+                "            return output\r\n",
+                "\r\n",
+                "def get_random_string(length):\r\n",
```

Comment:
I couldn't see where this method is being called from

Reply:
This was also in Aasim's SQLVM notebook and it wasn't used there either, it is kept here for consistency, unless it should be removed.